src/Microsoft.Extensions.Configuration.AzureAppConfiguration/PushNotification.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    /// <summary>
+    /// Object containing three parameters: A Uri subject, string SyncToken, and string EventType
+    /// for an App Configuration store.
+    /// </summary>
+    public class PushNotification
+    {			
+
+		/// <summary>
+		/// Uri Subject
+		/// </summary>
+		public Uri ResourceUri { get; set; }
+
+        /// <summary>
+        /// The Synchronization Token to be added to the next request to the App Configuration Service.
+        /// </summary>
+        public string SyncToken { get; set; }
+
+        /// <summary>
+        /// Event Type of Event Grid Message
```

Comment:
>Event Grid Message [](http://example.com/codeflow?start=26&length=18)

The PushNotification object is decoupled from Event Grid events. We could have a different event source in future, so we shouldn't talk about event grid (or its property names like 'subject') in this class. We should just summarize what this property is in AppConfig terms.
For example:

ResourceUri -> The URI of the key-value that triggered the push notification.

EventType -> The type of event that triggered the push notification.

Class summary -> An object containing the details of a push notification received from App Configuration service.